src/execution/operator/helper/physical_limit.cpp

```diff
@@ -13,28 +15,61 @@ class PhysicalLimitOperatorState : public PhysicalOperatorState {
 	idx_t current_offset;
 };
 
+uint64_t get_delimiter(DataChunk &input, Expression *expr) {
+	DataChunk limit_chunk;
+	vector<LogicalType> types{expr->return_type};
+	limit_chunk.Initialize(types);
+	ExpressionExecutor limit_executor(expr);
+	if (expr->type == ExpressionType::BOUND_FUNCTION) {
```

Comment:
Can't this branch be removed so we can always call `limit_executor.Execute(input, limit_chunk);`?